v19.2/create-table.md

```diff
@@ -103,140 +105,132 @@ In CockroachDB, every table requires a [primary key](primary-key.html). If one i
-| column_name | data_type | is_nullable | column_default | generation_expression | indices |
-+-------------+-----------+-------------+----------------+-----------------------+---------+
-| user_id     | INT       |    true     | NULL           |                       | {}      |
-| logon_date  | DATE      |    true     | NULL           |                       | {}      |
-+-------------+-----------+-------------+----------------+-----------------------+---------+
-(2 rows)
+  column_name | data_type | is_nullable | column_default | generation_expression |  indices  | is_hidden
++-------------+-----------+-------------+----------------+-----------------------+-----------+-----------+
+  id          | UUID      |    true     | NULL           |                       | {}        |   false
+  city        | STRING    |    true     | NULL           |                       | {}        |   false
+  name        | STRING    |    true     | NULL           |                       | {}        |   false
+  address     | STRING    |    true     | NULL           |                       | {}        |   false
+  credit_card | STRING    |    true     | NULL           |                       | {}        |   false
+  rowid       | INT8      |    false    | unique_rowid() |                       | {primary} |   true
+(6 rows)
 ~~~
 
 {% include copy-clipboard.html %}
 ~~~ sql
-> SHOW INDEX FROM logon;
+> SHOW INDEX FROM users;
 ~~~
 
 ~~~
+  table_name | index_name | non_unique | seq_in_index | column_name | direction | storing | implicit
 +------------+------------+------------+--------------+-------------+-----------+---------+----------+
-| table_name | index_name | non_unique | seq_in_index | column_name | direction | storing | implicit |
-+------------+------------+------------+--------------+-------------+-----------+---------+----------+
-| logon      | primary    |   false    |            1 | rowid       | ASC       |  false  |  false   |
-+------------+------------+------------+--------------+-------------+-----------+---------+----------+
+  users      | primary    |   false    |            1 | rowid       | ASC       |  false  |  false
 (1 row)
 ~~~
 
 ### Create a table (primary key defined)
 
-In this example, we create a table with three columns. One column is the [`PRIMARY KEY`](primary-key.html), another is given the [`UNIQUE` constraint](unique.html), and the third has no constraints. The `PRIMARY KEY` and column with the `UNIQUE` constraint are automatically indexed.
+In this example, we create the `users` table, but with some column [constraints](constraints.html). One column is the [`PRIMARY KEY`](primary-key.html), another is given the [`UNIQUE` constraint](unique.html), and another has both a `UNIQUE` constraint and a [`CHECK`](check.html) constraint that limits the length of the string. The `PRIMARY KEY` and column with the `UNIQUE` constraint are automatically indexed.
```

Comment:
I think our convention is to refer to these things in prose without upper-case or monospace as a "primary key", "unique constraint", "check constraint", etc., except in cases where we are directly referring to SQL statement text.  Although I was not able to find this in [our style guide](https://github.com/cockroachdb/docs/blob/master/STYLE.md), so maybe we need an update there.  But I'm pretty sure this is our convention based on convos I've had during PR reviews.

Reply:
Makes sense to me!

Reply:
Do we typically keep the reference links to other doc pages though?

Reply:
Yes - I am only referring to changing the text.